lib/rumbrl/ext/hash.rb

```diff
@@ -0,0 +1,8 @@
+# Extensions for native Hash to allow for log ingestion
+module HashExtensions
+  def to_log
+    to_hash
+  end
+end
+
+::Hash.include(HashExtensions)
```

Comment:
Seriously though, I don't have to do this. I can just like...make another branch in the conditional